Userland/Libraries/LibCore/TCPServer.cpp

```diff
@@ -64,6 +64,19 @@ bool TCPServer::listen(const IPv4Address& address, u16 port)
     return true;
 }
 
+void TCPServer::set_blocking(bool blocking)
+{
+    auto flags = fcntl(m_fd, F_GETFL);
```

Comment:
Missing error check.

Reply:
The only error check i can find for this elsewhere in the system is VERIFY(), is that the correct error check?